releaseNotes/master.md

```diff
@@ -9,9 +9,7 @@ ${RES_VER_DATE}
 
 
 ## Fixes
-  - **simple title**: brief description
```

Comment:
We should follow this format, add title and small description.
maybe: **Improve error logging**: Improved error logging for while fetching SHA for gitRepo resources during rSync processing